README.md

```diff
@@ -6,10 +6,83 @@
 
 ## Introduction
 
-Frequenz gRPC API for monitoring and control of microgrids.
+The Frequenz Microgrid API is an API for operating and monitoring electrical
+components and sensors within a microgrid. It offers various functionalities
+related to microgrid management, including metadata retrieval, component
+listing, setting power levels, and controlling the state of components. The API
+is structured around the concept of a microgrid, which is a localized energy
+system composed of various interconnected components and auxiliary sensors.
 
-This repository provides the core gRPC and protobuf spec together with
-support for generating dedicated API packages for various languages.
+The API supports different _categories_ of electrical components and sensors.
+Electrical components include batteries, inverters, meters, and more. Sensors
```

Comment:
I would suggest to list the currently supported categories as a bullet point list.

Reply:
I added links to the common API specs here.